cockroachdb/sqlalchemy/dialect.py

```diff
@@ -60,6 +60,7 @@ def __init__(self):
 
 class CockroachDBDialect(PGDialect_psycopg2):
     name = 'cockroachdb'
+    supports_comments = False
     supports_sequences = False
```

Comment:
Are we able to update this now (do we want to?)?

Reply:
Yeah, we probably can, but I'm just trying to get CI working again before enabling new features to test.